lib/chef/util/windows/logon_session.rb

```diff
@@ -47,7 +48,11 @@ def open
           password = wstring(original_password)
           domain = wstring(original_domain)
 
-          status = Chef::ReservedNames::Win32::API::Security.LogonUserW(username, domain, password, Chef::ReservedNames::Win32::API::Security::LOGON32_LOGON_NEW_CREDENTIALS, Chef::ReservedNames::Win32::API::Security::LOGON32_PROVIDER_DEFAULT, token)
+          if authentication == :remote
+            status = Chef::ReservedNames::Win32::API::Security.LogonUserW(username, domain, password, Chef::ReservedNames::Win32::API::Security::LOGON32_LOGON_NEW_CREDENTIALS, Chef::ReservedNames::Win32::API::Security::LOGON32_PROVIDER_DEFAULT, token)
```

Comment:
nitpick: It's hard to tell the difference between these two branches. I think it would be cleaner just to switch the changing input which looks like `LOGON32_LOGON_NEW_CREDENTIALS` vs `LOGON32_LOGON_NETWORK`